Tests/Controller/AbstractConnectControllerTest.php

```diff
@@ -35,47 +36,48 @@
 use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
 use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
 use Symfony\Component\Security\Core\User\UserCheckerInterface;
+use Symfony\Component\Security\Http\HttpUtils;
 use Symfony\Component\Templating\EngineInterface;
 
 abstract class AbstractConnectControllerTest extends TestCase
 {
     /**
-     * @var MockObject&AuthorizationCheckerInterface
+     * @var MockObject|AuthorizationCheckerInterface
      */
     protected $authorizationChecker;
 
     /**
-     * @var MockObject&TokenStorageInterface
+     * @var MockObject|TokenStorageInterface
      */
     protected $tokenStorage;
 
     /**
-     * @var MockObject&EngineInterface
+     * @var MockObject|EngineInterface
      */
     protected $twig;
 
     /**
-     * @var MockObject&RouterInterface
+     * @var MockObject|RouterInterface
      */
     protected $router;
 
     /**
-     * @var MockObject&ResourceOwnerMap
```

Comment:
wondering why most of these were changed? Seems for most of them `&` was correct and describes the type better than `|`

Reply:
This particular is in fact wrong... but some probably should be reverted, at some point phpstan started too moan & this change make him happy, TBH didn't debug why it moaned.